src/Migration/Step/PostProcessing/Model/ProductsInRootCatalog.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Migration\Step\PostProcessing\Model;
+
+use Migration\ResourceModel;
+
+/**
+ * Class can return products assigned to tree root category
+ */
+class ProductsInRootCatalog
+{
+    /**
+     * @var ResourceModel\Destination
+     */
+    private $destination;
+
+    /**
+     * @var array
+     */
+    private $productIds = null;
+
+    /**
+     * @var string
+     */
+    private $catalogCategoryProductDocument = 'catalog_category_product';
+
+    /**
+     * Id of category tree root
+     */
+    const TREE_ROOT_ID = 1;
```

Comment:
Why don't use private variable instead?
Isn't better to retrieve this *Id* instead hardcoding